pandas/tests/test_frame.py

```diff
@@ -5545,6 +5545,8 @@ def test_itertuples(self):
         dfaa = df[['a', 'a']]
         self.assertEqual(list(dfaa.itertuples()), [(0, 1, 1), (1, 2, 2), (2, 3, 3)])
 
+        self.assertEqual(type(next(df.itertuples(name=None))), tuple)
```

Comment:
Okay, reading that discussion I suspect your proposal is something like:
`self.assertNotIsInstance(<inst>, <namedtuple>)`

Please note that is impossible to check that <inst> is **not** a namedtuple this way (which I assume is what is to be tested):
* The caller of `itertuple()` has no access to the class created on-the-fly inside the method.
* It can surely never access it with `itertuple(name=None)` because it s not created in the first place. 
* The only common base class of different kinds of namedtuples is `tuple`.

With the test infrastrucure, I propose to change the line to:
`self.assertIs(type(<inst>), tuple)`

The only other possibility I can imagine would be something like `assert not hasattr(<inst>, '_fields')`.